text/353-cfn-registry-constructs.md

```diff
@@ -0,0 +1,467 @@
+# Constructs for Public CloudFormation Extensions
+
+* **Original Author(s):**: @eladb
+* **Tracking Issue**: #353
+* **API Bar Raiser**: @rix0rrr
+
+A set of construct libraries which includes generated constructs for all CloudFormation 
+resources and modules published to the public CloudFormation Registry.
+
```

Comment:
(Feeds into the next comment about repo org)

Can you add a minimal working backwards info on how users would build L2s on top of these L1s?

Perhaps in the form of contribution guide or a section in our "design guidelines".

Reply:
Not sure I understand what you are after...

Reply:
We usually build L2s on top of L1s. Are we going to encourage building L2s for these or are they going to remain as L1s. 
If so, where are these L2s going to be contributed to? i.e., which repo and how is it organized.

Reply:
We usually build L2s on top of L1s. 
Are we going to encourage building L2s for these or are they going to remain as L1s?

If so, where are these L2s going to be contributed to? i.e., which repo and how is it organized. 
Especially given that each type is its own module, and L2s (and bind patterns) bring multiple types together, how is this going to work?

Reply:
I don't believe people will build L2s for these. If they will, they can either use `cdk-import` and embed the imported L1s in the L2 library or just use these generated L1 libraries as a dependency of their L2s. At any case, this is out of scope here.